src/test/unit/motor_output_unittest.cc

```diff
@@ -28,6 +28,41 @@ extern "C" {
 #include "unittest_macros.h"
 #include "gtest/gtest.h"
 
+extern "C" {
+	uint8_t atomic_BASEPRI;
```

Comment:
Maybe we can get rid of this by including `#include "build/atomic.h"` in line 26?

Reply:
I tried but it throwed the error below

```
compiling ../main/drivers/dshot.c 
compiling unit/motor_output_unittest.cc 
linking ../../obj/test/motor_output_unittest/motor_output_unittest 
/usr/bin/ld: ../../obj/test/motor_output_unittest/drivers/dshot.c.o: en la función `__get_BASEPRI':
/home/blue/proyectos/betaflight/src/test/../main/build/atomic.h:51: referencia a `atomic_BASEPRI' sin definir
/usr/bin/ld: ../../obj/test/motor_output_unittest/drivers/dshot.c.o: en la función `__basepriRestoreMem':
/home/blue/proyectos/betaflight/src/test/../main/build/atomic.h:57: referencia a `atomic_BASEPRI' sin definir
/usr/bin/ld: ../../obj/test/motor_output_unittest/drivers/dshot.c.o: en la función `__basepriSetMemRetVal':
/home/blue/proyectos/betaflight/src/test/../main/build/atomic.h:64: referencia a `atomic_BASEPRI' sin definir
/usr/bin/ld: /home/blue/proyectos/betaflight/src/test/../main/build/atomic.h:64: referencia a `atomic_BASEPRI' sin definir
/usr/bin/ld: /home/blue/proyectos/betaflight/src/test/../main/build/atomic.h:65: referencia a `atomic_BASEPRI' sin definir
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make: *** [Makefile:777: ../../obj/test/motor_output_unittest/motor_output_unittest] Error 1
```

Reply:
tried

Reply:
fixed